config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
```

Comment:
No results found on staging, it's worth investigating whether they got scheduled in the lab.

Reply:
it could finish correctly a kselftest-filesystems job https://lava.ciplatform.org/scheduler/job/504728

Reply:
also futex https://lava.ciplatform.org/scheduler/job/498541

Reply:
OK: https://staging.kernelci.org/test/plan/id/617a60836c491a5207a6c47d/

However this platform is currently failing to boot (not a blocker for this PR):
https://staging.kernelci.org/test/case/id/618257cc859e62bcaea6c437/